public class ExceptionHandling3 {
    public static void main(String[] args) {
        // QUESTION 1
      /*   String str = null;
        System.out.println(str.length());
        //output : Runtime Exception  */

       //QUESTION 2
     /*    int a =10,b=0;
        try {
            int c= a/b;
            System.out.println(c);
        } catch (Exception e) {
            System.out.println(e);
        }
        //System.out.println(a/b); */

        // QUESTION 3
      /*   int arr[] = {1,2,3};
        try {
            System.out.println(arr[5]);
        } catch (Exception e) {
            System.out.println("Exception occured : " + e);
        }
        System.out.println("End of program"); */

        //QUESTION 7
       /*  try {
            String s = null;
            System.out.println(s.length());
        } finally{
            System.out.println("Finally block executed ");
        } */

        // QUESTION 5
        String s = null;
        System.out.println(s.length());
}
}
 
    